rpm/SPECS/iotasigfox.spec

```diff
@@ -0,0 +1,181 @@
+    /bin/mkdir -p _install_dir
+    echo "[INFO] Creating log & run directory"
+    /bin/mkdir -p %{_iotasigfox_log_dir}
+    chown -R %{_project_user}:%{_project_user} %{_iotasigfox_log_dir}
+    chown -R %{_project_user}:%{_project_user} _install_dir
+    chmod g+s %{_iotasigfox_log_dir}
+    setfacl -d -m g::rwx %{_iotasigfox_log_dir}
+    setfacl -d -m o::rx %{_iotasigfox_log_dir}
+
+    /bin/mkdir -p %{_iotasigfox_pid_dir}
+    chown -R %{_project_user}:%{_project_user} %{_iotasigfox_pid_dir}
+    chown -R %{_project_user}:%{_project_user} _install_dir
+    chmod g+s %{_iotasigfox_pid_dir}
+    setfacl -d -m g::rwx %{_iotasigfox_pid_dir}
+    setfacl -d -m o::rx %{_iotasigfox_pid_dir}
+
+    echo "[INFO] Configuring application service"
+    cd /etc/init.d
+    chkconfig --add %{_service_name}
+
+    # restores old configuration if any
+    [ -f /tmp/config.js ] && /bin/mv /tmp/config.js %{_install_dir}/config.js
+
+    # Chmod iotagent-sigfox binary
+    chmod guo+x %{_install_dir}/bin/%{_iotasigfox_executable}
+
+    echo "Done"
+
+# -------------------------------------------------------------------------------------------- #
+# pre-uninstall section:
+# -------------------------------------------------------------------------------------------- #
+%preun
+
+echo "[INFO] stoping service %{_service_name}"
+service %{_service_name} stop &> /dev/null
+
+if [ $1 == 0 ]; then
+
+  echo "[INFO] Removing application log files"
+  # Log
+  [ -d %{_iotasigfox_log_dir} ] && /bin/rm -rf %{_iotasigfox_log_dir}
+
+  echo "[INFO] Removing application run files"
+  # Log
+  [ -d %{_iotasigfox_pid_dir} ] && /bin/rm -rf %{_iotasigfox_pid_dir}
+
+  echo "[INFO] Removing application files"
+  # Installed files
+  [ -d %{_install_dir} ] && /bin/rm -rf %{_install_dir}
+
+  echo "[INFO] Removing application user"
+  userdel -fr %{_project_user}
+
+  echo "[INFO] Removing application service"
+  chkconfig --del %{_service_name}
+  /bin/rm -Rf /etc/init.d/%{_service_name}
+  echo "Done"
+fi
+
+# -------------------------------------------------------------------------------------------- #
+# post-uninstall section:
+# clean section:
+# -------------------------------------------------------------------------------------------- #
+%postun
+%clean
+/bin/rm -rf $RPM_BUILD_ROOT
+
+# -------------------------------------------------------------------------------------------- #
+# Files to add to the RPM
+# -------------------------------------------------------------------------------------------- #
+%files
+%defattr(644,%{_project_user},%{_project_user},755)
+%config /etc/init.d/%{_service_name}
+%attr(755, root, root) /etc/init.d/%{_service_name}
+%config /etc/logrotate.d/logrotate-iotasigfox.conf
+%config /etc/iotasigfox.d/iotasigfox.default.conf
+%config /etc/cron.d/cron-logrotate-iotasigfox-size
+%config /etc/sysconfig/logrotate-iotasigfox-size
+%config /etc/sysconfig/iotasigfox.conf
+%{_install_dir}
+
+%changelog
```

Comment:
As far as I understand, `%changelog` section has a predefined format that should be used. Have a look for instance to https://github.com/telefonicaid/fiware-orion/blob/master/rpm/SPECS/contextBroker.spec#L176

Reply:
Fixed in https://github.com/telefonicaid/sigfox-iotagent/pull/32/commits/b5e915f5bcbb0e2dbc99e9cab963460a477533d7